predicators/envs/cover.py

```diff
@@ -364,6 +378,36 @@ def _any_intersection(self,
         return False
 
 
+class CoverEnvHandEmpty(CoverEnv):
+    """Toy cover domain where the robot has a feature indicating whether its
+    hand is empty or not.
+
+    This allows us to learn all the predicates (Cover, Holding,
+    HandEmpty) through interactive learning.
```

Comment:
instead of saying "through interactive learning" let's say "with the assumption that the predicates are a function of only their argument's states"